docs/i18n/japanese/how-to-work-on-coding-challenges.md

```diff
@@ -73,27 +73,27 @@ assert.equal(
 );
 ```
 
-# --notes--
+# -- 注記 --
```

Comment:
@nhcarrigan As StephanDue points out in the comment on Crowdin, I think these (`--notes--` `--seed--` etc.) should not be translated. What do you think? 
https://freecodecamp.crowdin.com/translate/457d487b1586f10d2fba3c4ccb9c5453/5352/en-de/134#841821

Reply:
Correct, these should not be translated. This is part of the parser issue we discussed in chat.

Reply:
Understood. Thank you!